projects/qpid-proton/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder
+MAINTAINER jdanek@redhat.com
+RUN apt-get update && \
+    apt-get install -y \
+        cmake
+
+RUN git clone git://git.apache.org/qpid-proton.git
+
+WORKDIR qpid-proton
+COPY build.sh $SRC/
+COPY c_tests_fuzz_CMakeLists.patch $SRC/
```

Comment:
This is unfortunate; I hope you will eventually fix this upstream.

Reply:
Yes, should not take long. Linking -lc++ is trivial and building static library is already planned feature.